.github/workflows/CI.yml

```diff
@@ -350,22 +350,67 @@ jobs:
           key: "cached_image_${{ github.run_id }}"
           path: "${{ env.PACKAGING_ROOT }}"
 
-      - name: restore cached binaries
+      - name: Restore cached binaries
         if: steps.cached_image.outputs.cache-hit != 'true'
         uses: actions/cache@v2
         with:
           key: "cached_binaries_${{ github.run_id }}"
           path: build
 
-      - name: Build Image
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
```

Comment:
Why are we logging into the dockerhub? We aren't pushing the CI images there. Why not just login to the public ECR and build/push in one step?

Reply:
Yes we can do that. I am testing in different branch so this is not the final built yet. Still trying to figure it out how to pass data between jobs and steps at the same time so we do not need to use docker load or docker save.